### PR TITLE
feat: Add Release Health to wizard

### PR DESCRIPTION
**Pull request by HazAT**
This tells users to by default enable session health in the JS onboarding wizard.
Eventually, we want to make this the default behaviour but we decided to do this with a major release because it might upset some people if we sneak this into a minor bump.

This will give us some graceful way to get more customers using Release Health for JS.

_Also fixed backbone and angular1 docs._

### CODE REVIEW

**Comment by eurica** [comment]
@HazAT Could we get a screenshot for those of us less adept at parsing PRs? :)

**Comment by HazAT** [comment]
@eurica So it really only adds `autoSessionTracking: true` to the sample code in the onboarding wizard for JS/React/Angular/Gatsby. Making screenshots of that is 10x the work of the PR 😅

File: src/wizard/javascript/angular.md [hunk 1]
```diff
@@ -29,6 +29,7 @@ import { AppModule } from "./app/app.module";
 
 Sentry.init({
   dsn: "___PUBLIC_DSN___",
+  autoSessionTracking: true,
```
Comment (eurica) [inline src/wizard/javascript/angular.md:32]: Gotcha @HazAT, if this is what you're proposing, I'm strongly 👍

File: src/wizard/javascript/angularjs.md [hunk 2]
```diff
@@ -1,121 +1,45 @@
 ---
 name: AngularJS
-doc_link: https://docs.sentry.io/clients/javascript/integrations/angularjs/
+doc_link: https://docs.sentry.io/platforms/javascript/guides/angular/angular1/
 support_level: production
 type: framework
 ---
 
-### Installation
+To use Sentry with your Angular application, you will need to use `@sentry/angular` (Sentry’s Browser Angular SDK).
```
Comment (PeloWriter) [inline src/wizard/javascript/angularjs.md:8]:
```suggestion
Install our Browser Angular SDK using either `yarn` or `npm`:
```

File: src/wizard/javascript/angularjs.md [hunk 3]
```diff
@@ -1,121 +1,45 @@
...
-  "myAppFilters",
-]);
+# Using npm
+npm install --save @sentry/browser @sentry/integrations
 ```
 
-#### Module loaders (CommonJS) {#id1}
-
-The raven angular module can be loaded using a module loader like Browserify or Webpack.
+You should `init` the Sentry browser SDK as soon as possible during your application load up, before initializing Angular:
```
Comment (PeloWriter) [inline src/wizard/javascript/angularjs.md:20]:
```suggestion
`init` the Sentry Browser SDK as soon as possible during your page load, before initializing Angular:
```

File: src/wizard/javascript/angularjs.md [hunk 4]
```diff
@@ -1,121 +1,45 @@
...
+  autoSessionTracking: true,
+  integrations: [
+    new AngularIntegration(),
+    new Integrations.BrowserTracing({
+      tracingOrigins: ["localhost", "https://yourserver.io/api"],
+    }),
+  ],
+
+  // We recommend adjusting this value in production, or using tracesSampler
+  // for finer control
```
Comment (PeloWriter) [inline src/wizard/javascript/angularjs.md:39]: The model we've adopted removes the comments from the code, adding it as an explanatory text that follows. 
```suggestion

```

File: src/wizard/javascript/angularjs.md [hunk 5]
```diff
@@ -1,121 +1,45 @@
...
+  tracesSampleRate: 1.0,
+});
+
+// Finally require ngSentry as a dependency in your application module.
+angular.module("yourApplicationModule", ["ngSentry"]);
 ```
-
```
Comment (PeloWriter) [inline src/wizard/javascript/angularjs.md:120]: We recommend adjusting the value of `tracesSampleRate` in production. Learn more about configuring sampling in our [full documentation](https://docs.sentry.io/platforms/javascript/performance/sampling/).

Comment (PeloWriter) [reply src/wizard/javascript/angularjs.md:120]: We're also adding this to the end of each wizard (after the verification example):
If you're new to Sentry, use the email alert to access your account and complete a product tour.

If you're an existing user and have disabled alerts, you won't receive this email.

File: src/wizard/javascript/ember.md [hunk 6]
```diff
@@ -37,7 +37,7 @@ Then add the following config to your `config/environment.js`:
 ENV['@sentry/ember'] = {
   sentry: {
     dsn: '___PUBLIC_DSN___',
-
+    autoSessionTracking: true,
     // Set tracesSampleRate to 1.0 to capture 100%
     // of transactions for performance monitoring.
     // We recommend adjusting this value in production, or using tracesSampler
```
Comment (PeloWriter) [inline src/wizard/javascript/ember.md:43]: Take these comments out, adding the explanatory text at the end of the sample (replacing line 50)

"We recommend adjusting the value of `tracesSampleRate` in production. Learn more about configuring sampling in our [full documentation](https://docs.sentry.io/platforms/javascript/performance/sampling/)."

**Review by PeloWriter — APPROVED** [review]
Thank you.

**Pull request merged**